<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgUncomp</class>
 <widget class="QWidget" name="dlgUncomp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unpacked archive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget_Selection">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Standard_Button">
     <item>
      <widget class="themePushButton" name="pushButton_Ok">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/confirm.svgz</normaloff>:/icon/icons/confirm.svgz</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="themePushButton" name="pushButton_None">
       <property name="text">
        <string>None</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/window_close.svgz</normaloff>:/icon/icons/window_close.svgz</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>themePushButton</class>
   <extends>QPushButton</extends>
   <header>theme.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
